ci/environment.yml

```diff
@@ -14,6 +14,6 @@ dependencies:
   - pydap
   - pytest
   - pytest-cov
-  - shapely
+  - shapely<2.0
```

Comment:
Should this have been done in [the cf feedstock](https://github.com/conda-forge/xesmf-feedstock/blob/main/recipe/meta.yaml) as well?